### DISCUSSION TITLE
How to match CSS: font-size: larger?

### DISCUSSION THREAD

**Discussion by deerchao** [AvaloniaUI/Avalonia#12825] [Q&A] [answered]
<TextBlock>
      <Run />
      <Run />
    </TextBlock>

How do I make FontSize of the second Run larger, say 1.25 times its normal size( inherited from TextBlock I think).

**Comment by timunie** [comment]
Use a converter probably.

**Comment by maxkatz6** [comment, marked as answer]
Set absolute font size units or create resources for different font sizes style classes (like, h1, h2...).
